src/mate/src/Command/InitCommand.php

```diff
@@ -161,7 +161,7 @@ private function updateComposerJson(): array
 
         if (!isset($composerJson['extra']['ai-mate'])) {
             $composerJson['extra']['ai-mate'] = [
-                'scan-dirs' => ['mate'],
+                'scan-dirs' => ['mate/src'],
                 'includes' => ['config.php'],
```

Comment:
config file is relative to `mate/` but `scan-dirs` not? :thinking:

Reply:
yeah that seems a bit strange - should we also change this?